src/main/java/frc/robot/Robot.java

```diff
@@ -167,28 +168,24 @@ public void testInit() {
         CommandScheduler.getInstance().cancelAll();
     }
 
-    public CommandBase singulatorAndIntake() {
-        CommandBase cmd = new ParallelCommandGroup(intake.intake(), indexer.indexMotor(.85));
-        cmd.setName("Intake Balls");
-        return cmd;
-    }
-
     public CommandBase regurgitateFC() {
         CommandBase cmd = new ParallelCommandGroup(intake.discharge(), indexer.reversePush());
         cmd.setName("Regurgitate FC");
         return cmd;
     }
 
-    public CommandBase passLine() {
-        CommandBase cmd = new SequentialCommandGroup(drive.driveDistance(40, .5));
-        cmd.setName("Pass Line");
+    public CommandBase shootAuto() {
+        CommandBase cmd = new SequentialCommandGroup(shooter.spinUp(4500), indexer.shootAllBalls(3),
+                drive.driveDistance(20, .5));
+        cmd.setName("Shoot Auto");
         return cmd;
     }
 
     // will spin the shooter then shoot all the balls and then turn the shooter off.
     // TODO spin up is an instant command therefore it will never end
-    public CommandBase shootAllBalls() {
-        CommandBase cmd = new SequentialCommandGroup(shooter.spinUp(.8), indexer.shootAllBalls(), shooter.spinDown());
+    public CommandBase releaseAllBalls() {
+        CommandBase cmd = new SequentialCommandGroup(shooter.spinUp(4500), indexer.shootAllBalls(5),
+                shooter.spinDown());
```

Comment:
Remove `spinDown` here, and instead add it as a `whenReleased` on the same button